tests/cross_functional/system_test/test_mcg_replication_with_disruptions.py

```diff
@@ -404,3 +419,333 @@ def test_log_based_replication_with_disruptions(
+            noobaa_pods = get_noobaa_pods()
+            logger.info(
+                f"Updating object {object_key} with new version data in bucket {bucket_2.name}"
+            )
+            future = executor.submit(
+                self.upload_objects_with_retry,
+                mcg_obj_session,
+                bucket_2,
+                bucket_1,
+                mockup_logger_target,
+                test_directory_setup.origin_dir,
+                pattern=object_key,
+                prefix=prefix_2,
+            )
+            for pod_obj in noobaa_pods:
+                pod_obj.delete(force=True)
+                logger.info(f"Deleted noobaa pod {pod_obj.name}")
+            logger.info("Restarted all Noobaa pods")
+
+            # Wait for the upload to finish
+            future.result()
+
+            wait_for_object_versions_match(
+                mcg_obj_session,
+                awscli_pod_session,
+                bucket_1.name,
+                bucket_2.name,
+                obj_key=f"{prefix_2}/{object_key}",
+            )
+            logger.info(
+                f"Replication works from {bucket_2.name} to {bucket_1.name} "
+                f"and has all the versions of object {object_key}"
+            )
+            future.result()
+
+        # Take the noobaa db backup and then disable the sync versions
+        # make sure no version sync happens
+        logger.info("Taking backup of noobaa db")
+        noobaa_pvc_obj = get_pvc_objs(pvc_names=[constants.NOOBAA_DB_PVC_NAME])
+        _, snap_obj = noobaa_db_backup(noobaa_pvc_obj)
+
+        logger.info("Disabling version sync for both the buckets")
+        replication_1["rules"][0]["sync_versions"] = False
+        replication_2["rules"][0]["sync_versions"] = False
+
+        update_replication_policy(bucket_2.name, replication_1)
+        update_replication_policy(bucket_1.name, replication_2)
+
+        # Change the replication cycle delay to 3 minutes
+        logger.info("Reduce the bucket replication delay cycle to 5 minutes")
+        reduce_replication_delay(interval=5)
+
+        # Update previously uploaded object with new data and new version
+        self.upload_objects_with_retry(
+            mcg_obj_session,
+            bucket_1,
+            bucket_2,
+            mockup_logger_source,
+            test_directory_setup.origin_dir,
+            pattern=object_key,
+            prefix=prefix_1,
+            num_versions=4,
+        )
+        logger.info(
+            f"Updated object {object_key} with new version data in bucket {bucket_1.name}"
+        )
+
+        try:
+            wait_for_object_versions_match(
+                mcg_obj_session,
+                awscli_pod_session,
+                bucket_1.name,
+                bucket_2.name,
+                obj_key=f"{prefix_1}/{object_key}",
+            )
+        except TimeoutExpiredError:
+            logger.info(
```

Comment:
shouldn't this be error?

Reply:
Its a negative scenario where sync_versions=False. So it is expected to fail with timeout error as the versions dont sync